src/data/device_registry.ts

```diff
@@ -31,7 +31,7 @@ export const updateDeviceRegistryEntry = (
     ...updates,
   });
 
-const fetchDeviceRegistry = (conn) =>
+export const fetchDeviceRegistry = (conn) =>
```

Comment:
Please remove the export if you're not using it anymore.

Reply:
Fixed.